### PR TITLE
Updated the documentation of inverse Planck function for broad channels

### PR DESCRIPTION
**Pull request by rolle-at-work**
Updated the documentation of inverse Planck function for broad channels

 - [ ] Closes #xxxx 
 - [ ] Tests added 
 - [x] Tests passed: Passes ``pytest pyspectral`` 
 - [x] Passes ``flake8 pyspectral`` 
 - [x] Fully documented

### CODE REVIEW

**Comment by rolle-at-work** [comment]
Hmmm, can't be that complicated...

File: doc/rad_definitions.rst [hunk 1]
```diff
@@ -263,23 +263,82 @@ In python it may look like this:
    >>> print([round(t, 8) for t in temp])
    [299.99998562, 301.00000518]
 
+This approach only works for monochromatic or very narrow bands for which the 
+spectral response function is assumed to be constant. In reality, typical imager
+channels are not that narrow and the spectral response function is not contstant 
+over the band. Here it 
+is not possible to un-concatenate planck and spectral response function. That
```
Comment (adybbroe) [inline doc/rad_definitions.rst]: This needs better explanation. What is meant by
"un-concatenate planck and spectral response function"?

**Comment by adybbroe** [comment]
Excellent, thanks a lot @rolle-at-work !
I changed one word only. Let's wait till everything has passed again hopefully, and I will merge it!

**Comment by adybbroe** [comment]
LGTM

**Comment by mraspaud** [comment]
Reopening, and merging when everything passes

**Comment by adybbroe** [comment]
Yeah, sorry, I wasn't awake completely aparently!

**Pull request merged**